service/src/test/java/uk/gov/hmcts/reform/fpl/service/ManageDocumentServiceTest.java

```diff
@@ -56,11 +61,15 @@
     @Autowired
     private ManageDocumentService manageDocumentService;
 
+    @MockBean
+    private DocumentUploadHelper documentUploadHelper;
+
     private LocalDateTime futureDate;
 
     @BeforeEach
     void before() {
         futureDate = time.now().plusDays(1);
+        given(documentUploadHelper.getUploadedDocumentUserDetails()).willReturn("HMCTS");
```

Comment:
Tests should really be updated to assert `uploadedBy` has been added.

Reply:
i have added test data